src/Resources/translations/VerifyEmailBundle.pl.xlf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="file.ext" source-language="en" datatype="plaintext">
+        <body>
+            <trans-unit id="1">
+                <source>%count% year|%count% years</source>
+                <target>%count% roku|%count% lat</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>%count% month|%count% months</source>
+                <target>%count% miesiąca|%count% miesięcy</target>
```

Comment:
1 miesiąc
2 miesiące, 3 miesiące, 4 miesiące
5 miesięcy, 6 miesięcy and so on...

Reply:
done